pkg/pipelinecms/generate_name.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2021 Terminus, Inc.
```

Comment:
move pkg to modules/pipeline/pkg/cms.

dop can invoke pipeline's pkg.

Reply:
done